### PR TITLE
Add notes from call on 2015-02-11

### PR DESCRIPTION
**Pull request by amirmc**
I was unable to serve the site locally so these haven't been checked for any obvious errors/typos.

### CODE REVIEW

File: tmpl/wiki/weekly/2015-02-11.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+- Improving Quality
+- Error Handling RFC
+- TLS merge progress
+- OPAM 1.2 only
+- GSoC Projects
+- FOSDEM demo
+- Bitcoin Piñata
+
+
+Attendees: Amir Chaudhry, David Kaloper, Thomas Leonard, Jon Ludlam,
```
Comment (avsm) [inline tmpl/wiki/weekly/2015-02-11.md]: I was there! (briefly)

**Comment by avsm** [comment]
superb notes -- could become a blog post easily.

File: tmpl/wiki/weekly/2015-02-11.md (hunk 1 above)
Comment (amirmc) [inline tmpl/wiki/weekly/2015-02-11.md]: oops!

**Comment by amirmc** [comment]
Made a separate issue (#289) about my inability to serve locally.

File: tmpl/wiki/weekly/2015-02-11.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+test things in coordinated way for unikernels.  Specifically, we don't really
+have much end-to-end testing that looks at the complete toolchain. The only
+place where this surfaces for us is still mirage-www (this website!), as bugs
+tend to surface there whenever a wider issue occurs (cf. the recent crash on
+boot - [mirage/mirage#375][]). Finding bugs in an automated way and reporting
+them is important as the project grows.
+
+Some options include providing a manifest of specific versions and running
+tests that way.  It would allow us to take different libraries and bisect
+issues.  We would have to build this our as we still have very few automated
```
Comment (talex5) [inline tmpl/wiki/weekly/2015-02-11.md]: "this our as"?

File: tmpl/wiki/weekly/2015-02-11.md (hunk 1 above)
Comment (samoht) [inline tmpl/wiki/weekly/2015-02-11.md]: and me too!

Comment (amirmc) [inline tmpl/wiki/weekly/2015-02-11.md]: Done (and sorry!). Next time we don't leap in until I've done a proper roll call. :)

**Comment by amirmc** [comment]
I'm happy for this to be merged (I'll merge myself this afternoon if no-else does).  

> superb notes -- could become a blog post easily

They do go out via the [updates atom feed](http://openmirage.org/updates/atom.xml), which is good (that's just a PSA for anyone watching). A post on our test infrastructure would awesome and the notes here cover the background motivation.  I've added an item to next week's agenda about test infra. Might make this a rolling item (i.e every call or every other call).

**Comment by amirmc** [comment]
Is there any way to get an automated notice when the new unikernel is deployed?

**Comment by mor1** [comment]
when it is actually booted? could put something in the deploy script
perhaps; or setup a cohttp callback in teh unikernel to ping a url?

On 18 February 2015 at 22:00, Amir Chaudhry <notifications@github.com>
wrote:

> Is there any way to get an automated notice when the new unikernel is
> deployed?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/mirage/mirage-www/pull/288#issuecomment-74957362>.
>



-- 
Richard Mortier
mort@cantab.net

**Comment by amirmc** [comment]
That's kinda what I was thinking. Specifically in two steps. 

1. A notification when deployable Unikernel is committed to repo. This is just a GitHub post-commit hook, when Travis pushes to the repo. 

2. A notification when new Unikernel is booted. This could either come from (a) the Unikernel itself (on boot -> ping a URL), or (b) from dom0. If from Unikernel, you'll only get notifications of successful boot but might miss a failure. From dom0, you'd be able to send a note if the boot failed. Having something like this would be useful for automating other bits of infrastructure e.g. new VM is live -> switch over DNS (or something). 

I'll send this to the mailing list as others might have thoughts on it. In my case, I just want to know when the minutes are live :)


--
sent via mobile

**Pull request merged**